### PR TITLE
Update the Your First App document to match current SDK

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: Develop/Apps/Your_First_App/README.md [hunk 1]
```diff
@@ -34,15 +34,15 @@ The SDK comes with a Sailfish UI template project that makes it very easy to get
          class="md_thumbnail" style="max-width:100%"/>
 </a>
 
-2\. Select **Applications→Sailfish OS Qt Quick Application** and then click **Choose**.
+2\. Select **Applications→Sailfish OS Qt Quick Application** and then click **Choose**. Alternatively, if you want to create a QML only project, select **Applications→Sailfish OS Qt Quick Application (QML Only)**.
```
Comment (jovirkku) [inline Develop/Apps/Your_First_App/README.md:37]: The template selector of the related picture has the word "Application" (i.e., singular) while the text has "Applications ->"

**Review by jovirkku — COMMENTED** [review]
Overall, I can see a considerable improvement in the clarity and details of the modified sentences. Also, the language is better. Good work.

Nitpicking: 
- line 37
- step "6. Click Finish.":  I wonder if picture "QtC_Template_04.png" should be renewed as it shows a different project location (not "/home/vige" any more but "/home/jare") which may be confusing for a person reading this doc the first time.
(yes, there were no changes to step 6 but I was viewing the whole doc at https://github.com/sailfishos/docs.sailfishos.org/blob/67de4df2584f046154f31b41c51fa920bb1bf8e9/Develop/Apps/Your_First_App/README.md)

File: Develop/Apps/Your_First_App/README.md (hunk 1 above)
Comment (vigejolla) [reply Develop/Apps/Your_First_App/README.md:37]: Good catch, thanks!

**Comment by vigejolla** [comment]
> Overall, I can see a considerable improvement in the clarity and details of the modified sentences. Also, the language is better. Good work.
> 
> Nitpicking:
> 
>     * line 37
> 
>     * step "6. Click Finish." in [this chapter](https://github.com/sailfishos/docs.sailfishos.org/blob/67de4df2584f046154f31b41c51fa920bb1bf8e9/Develop/Apps/Your_First_App/README.md#create-a-sailfish-application-project):  I wonder if picture "QtC_Template_04.png" should be renewed as it shows a different project location (not "/home/vige" any more but "/home/jare") which may be confusing for a person reading this doc the first time.
>       (yes, there were no changes to step 6 but I was viewing the whole doc at https://github.com/sailfishos/docs.sailfishos.org/blob/67de4df2584f046154f31b41c51fa920bb1bf8e9/Develop/Apps/Your_First_App/README.md)

I suppose it makes sense. Also because the list of files has changed, e.g. we no longer have yaml which is visible in the old picture.

**Comment by vigejolla** [comment]
> > Overall, I can see a considerable improvement in the clarity and details of the modified sentences. Also, the language is better. Good work.
> > Nitpicking:
> > ```
> > * line 37
> > 
> > * step "6. Click Finish." in [this chapter](https://github.com/sailfishos/docs.sailfishos.org/blob/67de4df2584f046154f31b41c51fa920bb1bf8e9/Develop/Apps/Your_First_App/README.md#create-a-sailfish-application-project):  I wonder if picture "QtC_Template_04.png" should be renewed as it shows a different project location (not "/home/vige" any more but "/home/jare") which may be confusing for a person reading this doc the first time.
> >   (yes, there were no changes to step 6 but I was viewing the whole doc at https://github.com/sailfishos/docs.sailfishos.org/blob/67de4df2584f046154f31b41c51fa920bb1bf8e9/Develop/Apps/Your_First_App/README.md)
> > ```
> 
> I suppose it makes sense. Also because the list of files has changed, e.g. we no longer have yaml which is visible in the old picture.

updated

**Review by jpwalden — APPROVED** [review]
LGTM, Clear instructions with good screenshots.

**Comment by jovirkku** [comment]
Looks good now.

**Review by rainemak — APPROVED** [review]

**Pull request merged**